tests/schema.py

```diff
@@ -39,6 +39,26 @@ class Subject(dj.Manual):
     def prepare(self):
         self.insert(self.contents, ignore_errors=True)
 
+@schema
+class Language(dj.Lookup):
+
+    definition = """
+    # languages spoken by some of the developers
+
+    entry_id    : int
+    ---
+    name        : varchar(40) # name of the developer
```

Comment:
Sorry, this table is not in 3rd normal form.   I know it's just a test example but we might as well show good examples.